app/js/services/v3Properties.json

```diff
@@ -645,6 +645,16 @@
         "children": [
           {}
         ]
+      },
+      "airplay": {
+        "label": "AirPlay",
+        "type": "featuremenu",
+        "description": "Checking this option specifies whether the player should include the AirPlay plugin or not.",
+        "model": "config.plugins.airplay",
```

Comment:
why airplay it's not under `cast` config?

Reply:
AirPlay it's a plugin, cast is a framework.

Reply:
this is something only you know...
For app, I guess, it's not relevant. airplay it's like cast in the functional aspect. also, they are under Cast in the studio so it's confusing

Reply:
Confusing to whom?
Who says that the UI should reflect to configuration structure?
In the description we are giving the fact that airplay is a plugin a reference.
I don't think airplay should be under cast config since it's not using the cast framework.
BTW, same as `ima` not under `advertising` config, they are the same in the functional aspect. Why is that?